projects/ngx-formentry/src/components/ngx-remote-select/remote-select.component.html

```diff
@@ -13,6 +13,7 @@
     typeToSearchText="Please enter 2 or more characters"
     [typeahead]="remoteOptionInput$"
     [(ngModel)]="selectedRemoteOptions"
+    [appendTo]="'form'"
```

Comment:
Without this append, the dropdown does not show on top of the other content, I tried removing it and the dropdown was appearing behind the form. That's the solution I got that will solve the issue.

Reply:
To answer you elaborately and why I used the `appendTo`, 
```
The [appendTo] attribute on an ng-select directive is used to 
specify where the dropdown menu of the select should be 
appended to in the DOM. By default, the dropdown menu is appended to the body of the document, 
which means it will appear on top of all other content on the page.

However, in some cases, you may want to append the dropdown menu 
to a specific element in the DOM, such as a form or a container div. 
This is where the [appendTo] attribute comes in.

When you set [appendTo] to 'form', you are telling the ng-select 
directive to append the dropdown menu to the nearest ancestor 
form element in the DOM. 
This can be useful if you want to ensure that the dropdown menu i
s always contained within the form element, and doesn't appear outside of it.
```

And this is the solution I got to ensure that our dropdown panel appears above the content.
But I am open to a better solution. cc @ibacher

Reply:
Yes, but see the larger note I wrote as a summary. _If_ this is the part doing the work (and it is), then we don't need the code changes in `ngx-remote-select.component.ts`. However, that also means that this isn't _just_ fixing drop-downs at the end of the form. Now _every_ remote-select component will render the drop-down elements as a sibling of the `form` element. That also means that in the css file, we only really need _one_ rule (because there is never a component that will match `.ng-select > .ng-dropdown-panel`). So basically, this PR does too many things, most of which aren't actually used.

An alternative implementation would be to drive this `appendTo` property from some kind of property in the component, in which case the code in `ngx-remote-select.component.ts` and the css file would make sense, but then this property would need to point to the component property and _not_ a fixed selector.

Reply:
Actually, the other selects which are not remote get changed by the negative margin I add to the remote select and they look weird, that's why I also need to target their dropdown panel with a 0 margin-top.

Reply:
@ibacher I have removed all the implementation in the `ngx-remote-select.component.ts`, the dropdowns are fixed in the `ngx-form-entry` forms but they are not fixed in the patient chart. I am not sure why the fix is not working for the patient chart 🤔 .

Reply:
I just realised the the `appendTo` is not reflecting in the patient chart but if I add it manually the dropdown works, maybe the linking is the problem.

Reply:
@ibacher @rbuisson I have tested the just the `appendTo` change and it is also working for the patient chart.